src/pages/docs/creators/ray/setup-tutorial.md

```diff
@@ -0,0 +1,156 @@
+---
+description: step-by-step explanation of the quickstart instructions
+title: Ray on Golem setup tutorial
+type: guide 
+---
```

Comment:
This `md` has the same issues as `[quickstart.md](https://github.com/golemfactory/golem-docs/pull/74/files#diff-ef1f3b84fb79f3529f1da60aa7bb69cc595e01ce31932bbc9f286ca52ac323b9)`